onnx/defs/function.h

```diff
@@ -141,6 +141,22 @@ class FunctionBuilder {
     return Add(node_txt, MakeAttribute(attr_name, attr_value));
   }
 
+  template <typename... Args>
+  FunctionBuilder& Add(const char* node_txt, Args... args) {
+    if constexpr (sizeof...(args) % 2 == 0) {
+      return AddAttributes(node_txt, args...);
+    }
+  }
+
+  template <typename T, typename... Args>
+  FunctionBuilder& AddAttributes(const char* node_txt, const std::string& attr_name, const T& attr_value, Args... args) {
+    Add(node_txt, MakeAttribute(attr_name, attr_value));
+    if constexpr (sizeof...(args) > 0) {
+      AddAttributes(node_txt, args...);
```

Comment:
Has this been tested? Is it used in this PR? It doesn't look right. The node corresponding to `node_txt` will be added multiple times here, won't it?